app/react/docker/DashboardView/ContainerStatus.tsx

```diff
@@ -17,37 +17,21 @@ export function ContainerStatus({ containers }: Props) {
     <div className="pull-right">
       <div>
         <div className="vertical-center space-right pr-5">
-          <Icon
-            icon="power"
-            className={clsx('icon icon-sm icon-success')}
-            feather
-          />
+          <Icon icon="power" className={clsx('icon icon-sm icon-success')} />
```

Comment:
looks like clsx is not needed in this file

Reply:
updated